File: PHP/3 Ano/20-03-10/dart/main.dart
import 'dart:io';

main() {
    print('Seu nome: ');
    var name = stdin.readLineSync();

    print('Quantida de vezes para repetirrepetir: ');
    var number = int.parse(stdin.readLineSync());


    for (var i = 0; i < number; i++) {
      print(name);
    }



}

File: PHP/3 Ano/20-03-10/pascal/main.pas
program NicePisco;
uses crt;

var myName: String;
var number, i: Integer;

begin
   writeln('Seu Nome: ');
   readln(myName);

   writeln('Quantidade de vezes para repetir: ');
   readln(number);

   for i:= 1 to number do begin
      writeln(myName);
   end;
  
end.
